notebooks/Section1_2-Hierarchical_Models.ipynb

```diff
@@ -56,7 +56,7 @@
   },
```

Comment:
<p>Why did you remove the exercise? Even if we skip participants might do them on their own.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fonnesbeck/Bayes_Computing_Course/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='76'/>

Reply:
<p>I'm using it in the homework.</p>